### PR TITLE
tests: fix flaky integration test

### PR DESCRIPTION
**Pull request by pazembrz**
Quick fix for flaky integration test

### CODE REVIEW

File: backend/tests/integration/records/test_utils.py [hunk 1]
```diff
@@ -94,7 +94,8 @@ def test_get_parent_record_when_more_than_one(inspire_app):
     }
     rec = create_record("lit", data=data)
     extracted_parent_record = get_parent_record(rec)
-    assert extracted_parent_record == parent_record
+    parent_records = list(parent_record, second_parent_record)
```
Comment (drjova) [inline backend/tests/integration/records/test_utils.py:97]: do I miss something or this should throw an exception? :/

Comment (pazembrz) [reply backend/tests/integration/records/test_utils.py:97]: Why do you think that?

Comment (drjova) [reply backend/tests/integration/records/test_utils.py:97]:
```
----> 1 list('a', 'b')

TypeError: list() takes at most 1 argument (2 given)
```

Comment (pazembrz) [reply backend/tests/integration/records/test_utils.py]: Ah, yest... Sorry... Monday morning....

**Pull request merged**